src/styleguide/components/s-api-mock-test.vue

```diff
@@ -273,26 +273,25 @@
               // Do nothing
             }
 
-            const resolver: ResponseResolver = (req, res, ctx: any) => res( // eslint-disable-line @typescript-eslint/no-explicit-any
-              ctx.status(parseInt(configuration.status, 10)),
-              response ? ctx.json(response) : null
-            );
+            const resolver: ResponseResolver = () => HttpResponse.json(response as object, {
```

Comment:
suggestion: I would also recommend to change this. You could already set the type when defining the component (`let response = {}`). I think this would make it also clear, that `configuration.response` has to be of this type.